src/services/topicInstance/hooks.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+const globalHooks = require('../../hooks');
+const auth = require('feathers-authentication');
```

Comment:
Codacy found an issue: [`feathers-authentication` import should occur before import of `../../hooks`](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045819)